### ISSUE TITLE
add_issues_to_epic not working - API changed?

### ISSUE THREAD

**Issue by xyxz-web** [pycontribs/jira#964]
I'm trying to add an issue to an epic by calling add_issues_to_epic:
```
epic_id = int(jira.issue(epic.key).id)
jira.add_issues_to_epic(epic_id, [issue.key])
```
However, this leads to the process freezing for a while and then failing.

If I interpret https://developer.atlassian.com/cloud/jira/software/rest/api-group-epic/#api-agile-1-0-epic-epicidorkey-issue-post correctly, the API might have changed.

pycontribs jira sets the issue keys like that:
```data = {}
data['issueKeys'] = issue_keys
```

While above site sets them like this:
```
# This code sample uses the 'requests' library:
# http://docs.python-requests.org
import requests
import json

url = "https://your-domain.atlassian.net/rest/agile/1.0/epic/{epicIdOrKey}/issue"

headers = {
   "Content-Type": "application/json"
}

payload = json.dumps( {
  "issues": [
    "<string>"
  ]
} )

response = requests.request(
   "POST",
   url,
   data=payload,
   headers=headers
)

print(response.text)
```

**Comment by gredin** [comment]
https://ecosystem.atlassian.net/browse/ACJIRA-1634 points to the new method:
> Move issues to epic - Edit the issue  and set the parent field. Example: {"fields":{"parent":{"id":"11111"}}}

And here's my implementation:
```from jira import JIRA

jira = JIRA('https://ABC.atlassian.net', basic_auth=('EMAIL', 'TOKEN'))

epic = jira.issue('XYZ-47')

new_issue = jira.create_issue(
    project='XYZ',
    summary='New task',
    description='This new task is about stuff.',
    issuetype={'name': 'Task'}
)

new_issue.update(fields={'parent': {'id': epic.id}})
```

**Comment by adehad** [comment]
Should be fixed in 3.3.0 via #1351
